func/utilities.c

```diff
@@ -17,10 +18,10 @@ void flushInput(){
 /**
  * Capitalises the first character of a string.
  */
-void capitaliseFirst(char* string){
+char* capitaliseFirst(char* string){
```

Comment:
I bruger `toLowerCase` og `capitaliseFirst` som om de returnerer en ny string med nogle ændringer, men det er ikke det der sker. I virkeligheden laver de om i den originale streng og returnerer en pointer til den originale streng. Derfor fjernede jeg muligheden for at bruge funktionen som en direkte return statement, så det vil være tydeligt at det er den originale streng der bliver manipuleret, og så man ikke vil komme til at lave den fejl I har lavet i `ui.c:128`. Se mere i #38